js/PageSize.js

```diff
@@ -0,0 +1,11 @@
+function pgsize(value) {
+  if (value == 'notebook') {
+    document.getElementById('page-a').style.height = (11 * 400) / 8.5;
```

Comment:
Hi I guess you're supposed to add `+ 'px'` at the end to get that value in pixels

Reply:
oh and page-a is class so you will have to replace `document.getElementById('page-a')` with `document.querySelector('.page-a')`